server/src/routes/api/v1/parksReviewRouter.js

```diff
@@ -0,0 +1,31 @@
+import express from "express"
+import { Review } from "../../../models/index.js"
+import objection from "objection"
+import { ValidationError } from "objection"
+import cleanUserInput from "../../../services/cleanUserInput.js"
+import ReviewSerializer from "../../../db/serializers/ReviewSerializer.js"
+
+const parksReviewRouter = new express.Router({ mergeParams: true })
+parksReviewRouter.post("/", async (req, res) => {  
+    try {
+        const { body } = req
+        const cleanedInput = cleanUserInput(body.review)
+        const userId = req.user.id
+        body.review.userId = userId
+        const { id } = req.params
+        body.review.dogParkId = id
+        const review = await Review.query().insertAndFetch(cleanedInput)
```

Comment:
the request `body` should not be modified - I'm also not sure how the Review is created with the correct info as it is passed `cleanedInput`, but the `userId` and `dogParkId` are attached to the `body` object? `cleanedInput` should contain all necessary properties for the insertAndFetch